nf-test.config

```
@@ -1,19 +1,23 @@
 config {
-    // location for all nf-tests
-    testsDir "tests"
+    // location for all nf-test tests
+    testsDir "."
 
     // nf-test directory including temporary files for each test
-    workDir ".nf-test"
-
-    // location of library folder that is added automatically to the classpath
-    libDir "tests/pipeline/lib/"
+    workDir System.getenv("NFT_WORKDIR") ?: ".nf-test"
 
     // location of an optional nextflow.config file specific for executing tests
     configFile "tests/nextflow.config"
 
-    // run all test with the defined docker profile from the main nextflow.config
-    profile "docker"
+    // ignore tests coming from the nf-core/modules repo
+    ignore 'modules/nf-core/**/*', 'subworkflows/nf-core/**/*'
```

```suggestion
    ignore 'modules/nf-core/**/tests/*', 'subworkflows/nf-core/**/tests/*'
```
another tiny bug fixed in the next template update https://github.com/nf-core/tools/pull/3609/files

---

why did you resolve but not apply this comment? 🤔